namespaces/Representor.md

```diff
@@ -0,0 +1,44 @@
+# Representor Namespace
+
+This document extends [Refract](../refract-spec.md) Specification to describe the [Representor](https://github.com/the-hypermedia-project/charter) DOM.
+
+## Resource (Element)
+
+The Resource represents a Hypermedia Resource, with its available transitions and attributes.
+
+### Properties
+
+- `element`: resource (string, fixed)
+- `attributes` (object)
+    - `transitions` - (array[Transition]) - The available transitions from this resource.
+    - `resources` - (array[Resource]) - Any embedded (resources) from this resource.
+    - `relation` - (string) - An optional relation to this resource, applicable when embedded in another resource.
+- `content` (object) - Any properties of the resource.
+
+## Transition (Object Element)
+
+A transition is an available progression from one state to another state.
+
+### Properties
+
+- `element`: transition (string, fixed)
+- `attributes` (object, required)
+    - `relation` - (string, required) - The relation of this transition from the current resource.
+    - `parameters` ([Object Type][]) - A collection of parameters for this transition.
+    - `attributes` ([Object Type][]) - A collection of possible attributes for this transition.
```

Comment:
> First, why the MSON Object Type instead of the Refract Object Element? I have felt for a while we've been putting MSON things into the base Refract spec, and even brought up whether the entirety of MSON should be in the base spec. Using the MSON Object Type here makes it seem to me like something is off, either the way we've defined these specs or the way this is being used.

You MAY need a description of data structure here. That is why. Not just plain data. If we put MSON element here it MAY be just plain data (Element) or data description (MSON Element)